src/test/article.spec.js

```diff
@@ -213,3 +219,246 @@ describe('GET api/v1/articles', () => {
       });
   });
 });
+describe('POST/GET/PUT/DELETE ACTIVITIES ON ARTICLES CONTROLLER', () => {
+  const token = helper.jwtSigner({
+    id: 2,
+    username: 'Grace',
+    email: 'grace@grace.com'
+  });
+
+  describe('POST /api/v1/articles', () => {
+    beforeEach(async () => {
+      await db.Bookmark.destroy({ cascade: true, truncate: true });
+      await db.User.destroy({ cascade: true, truncate: true });
+      await db.Article.destroy({ cascade: true, truncate: true });
+    });
+    it('should create article successfully with valid user input', async () => {
+      const response = await chai
+        .request(app)
+        .post(ARTICLES_API)
+        .set('x-access-token', token)
+        .send(articleSample);
+      expect(response.status).to.equal(201);
+      expect(response.body.data).to.have.property('id');
```

Comment:
Do remove the property assertions when there are accompanying value assertions

Reply:
Thank you for this feedback. 
I have resolved them.